src/main/java/net/greatstart/dto/DtoProjectDescription.java

```diff
@@ -7,12 +7,11 @@
 import javax.validation.constraints.Size;
 import java.math.BigDecimal;
 import java.time.LocalDate;
-import java.util.Arrays;
 
 @Data
 public class DtoProjectDescription {
     @NotNull
-    @Size(min = 3, max = 35, message = "{name.size}")
+    @Size(min = 3, max = 100, message = "{name.size}")
     private String name;
     private String goal;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "goal" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)